src/shared/ui/NavBack/NavBack.tsx

```diff
@@ -0,0 +1,23 @@
+'use client';
+import React from 'react';
+import { useRouter } from 'next/navigation';
+
+import { Icon } from '../Icon';
+
+type NavBackProps = {
```

Comment:
```suggestion
type Props = {
```
convention 으로 변경이 필요해 보입니다 !

Reply:
Props로 변경 반영하였습니다 !